demo-app/src/pages/default/index.jsx

```diff
@@ -9,6 +9,18 @@ import {
   ReverseAnchor,
 } from '../../components';
 
+const tileLayer = {
+  // url: 'https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}{r}.png',
+  // attribution: `
+  //   &copy; <a href="https://stadiamaps.com/">Stadia Maps</a>,
+  //   &copy; <a href="https://openmaptiles.org/">OpenMapTiles</a>
+  //   &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors`,
+  format: 'pmtiles',
+  url: 'http://localhost:8080/planet/{z}/{x}/{y}.mvt',
```

Comment:
Does someone testing the demo-app have to do anything special to get this server running locally? or does it automatically start with the `dev` script?

Just wondering from a dev experience of someone looking at the project.

Reply:
Modified comment to address as a temp fix